### PR TITLE
Adding basic support for VSCode

### PR DESCRIPTION
**Pull request by adam-becker**
Not sure if these files are desired in this repo or not, but this PR provides basic support for spawning the `erlang_ls` process and connecting to it from VSCode.

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "dockerFile": "Dockerfile",
+  "extensions": [
+    "pgourlain.erlang"
```
Comment (robertoaloi) [inline .devcontainer/devcontainer.json]: What is this extension?

**Comment by robertoaloi** [comment]
Hi @adam-becker and thanks for the contribution! This looks promising, but I would probably use the same approach that we used for the Emacs stuff. I.e. having this in a separate repository and add instructions to the README on how to integrate `erlang_ls` with `VS Code`. What do you think?

**Comment by robertoaloi** [comment]
And I'm happy to have it as a separate repo under the `erlang-ls` organization or your own repo, as you prefer.

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (adam-becker) [reply .devcontainer/devcontainer.json]: It's an extension we've used at work, it provides some very basic language support. Syntax highlighting, and some auto-complete.

**Comment by adam-becker** [comment]
I'm fine with implementing this as a stand-alone repo under the `erlang-ls` organization. I like including the server with the extension, so I'll probably have to use a git submodule to reference this repo. It would be a good way of making sure bleeding edge features don't get pushed to the extension until they're ready.

**Comment by robertoaloi** [comment]
I created the repository and named it `vscode`. Go wild and let me know if you would like to be named something different! Please add a VSCode section to the `erlang_ls` README. I promise I will try the instructions :)

**Comment by adam-becker** [comment]
Thanks! I'll take a look at it tonight.

**Comment by robertoaloi** [comment]
@adam-becker So are you going to decline this PR and open it against the vscode repo, instead?

**Comment by adam-becker** [comment]
> @adam-becker So are you going to decline this PR and open it against the vscode repo, instead?

Yes, I've had less free time lately but I plan to work on this soon.

**Comment by robertoaloi** [comment]
@adam-becker Nice, thanks. I created an empty README in the `vscode` repo, which should now be _forkable_. I will close this PR for now.

**Pull request closed without merging**